db/migrate/20170112142045_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,15 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[5.0]
+  def up
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_index  :users, :confirmation_token, unique: true
+    add_column :users, :unconfirmed_email, :string
+    execute("UPDATE users SET confirmed_at = NOW()")
```

Comment:
I think it would confuse users who come back and don't expect this. There is no easy way for those users to understand what is wrong and why they now need to confirm their accounts when those accounts are years old.